README.md

```diff
@@ -95,25 +95,25 @@ This short example assumes Consul is installed locally.
   $ consul agent -dev
   ```
 
-1. Author a template `in.tpl` to query the kv store:
+2. Author a template `in.tpl` to query the kv store:
 
   ```liquid
   {{ key "foo" }}
   ```
```

Comment:
These code blocks need to be indented by one more space in order for the markdown parser to correctly recognize that they belong to the parent list item and retain the sequential numbering when rendering the doc.

```suggestion
   ```liquid
   {{ key "foo" }}
   ```
```